crates/bevy_reflect/src/impls/std.rs

```diff
@@ -996,252 +995,14 @@ impl<T: Reflect + TypePath + GetTypeRegistration, const N: usize> GetTypeRegistr
     }
 }
 
-impl<T: FromReflect + TypePath + GetTypeRegistration> GetTypeRegistration for Option<T> {
```

Comment:
Most of the conflicts were from the `+ GetTypeRegistration` which didn't exist before